release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1313,6 +1313,19 @@ sourceStrategy:
 
+* Previously, CSR renewal was handled by `kube-controller-manager` and correctly left pending by the machine approver, which increased `mapi_current_pending_csr` to `1`. Afterwards, the `kube-controller-manager` approved the CSR, but the machine approver would ignore it, which left the metric unchanged. Consequently, the `mapi_current_pending_csr` was stuck at `1` until another machine approver reconciled it. With this update, CSR approvals are reconciled from other controllers to update metrics properly. As a result, `mapi_current_pending_csr` is always up-to-date after every reconcile. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072195[*BZ#2072195*])
+
+* With {product-title} {product-version}, if an insufficient number of worker nodes start upon cluster installation, the Machine API Operator is now reported as `degraded` and errors are posted in the installation log. Previously, the Machine API Operator did not report as `degraded` in this scenario, even though other Operators were reported as `degraded`. As a result, with the Machine API Operator in the list of failed Operators, users now know to examine the state of the machines as the reason why there are insufficient worker nodes.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1994820[*BZ#1994820*])
+
+* Because the Machine API Operator did not honor the proxy environment variable directives, installation behind an HTTP or HTTPS proxy failed. With this fix, the HTTP transport logic of Machine API Operator now obeys proxy directives. As a result, the Machine API Operator now works in a restricted environment where egress traffic is only allowed through a proxy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082667[*BZ#2082667*])
+
+* Installations on vShpere for clusters with thousands of tags and heavy API loads were failing. Now the machine controllers only query tags that are related to a particular OCP installation. As a result, {product-title} installs correctly on vSphere for these clusters. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2097153[*BZ#2097153*])
+
+* Because the kubelet needs to contact the vCenter to obtain node zone labels, if the vCenter credentials are stored in a secret, the kubelet could not start because the kube client was not created on time. As a reault, if you rebooted the nodes, as happens when you edit the `cloud-provider-config` config map, the nodes did not come up. With this fix, the kubelet now obtains zone labels after node registration if the credentials stored in secret. As a result, the nodes reboot as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1902307[*BZ#1902307*])
```

Comment:
```suggestion
* Because the kubelet needs to contact the vCenter to obtain node zone labels, if the vCenter credentials are stored in a secret, the kubelet could not start because the kube client was not created on time. As a result, if you rebooted the nodes, as happens when you edit the `cloud-provider-config` config map, the nodes did not come up. With this fix, the kubelet now obtains zone labels after node registration if the credentials are stored in the secret. As a result, the nodes reboot as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1902307[*BZ#1902307*])
```